### ISSUE TITLE
Restrict Access is showing no options with this Theme

### ISSUE THREAD

**Issue by giuliogatto** [willianmano/moodle-theme_moove#185]
Hello, when switching to this theme there is a weird behaviour in Activity->Edit Settings.
When trying to add a Restrict Access option to any activity no options appear.
(see attached screenshot)
If I revert back to theme Classic then the options appear, so I guess this is a theme related problem. Could it be a bug?
Thanks
![Screenshot 2019-10-17 at 10 17 19](https://user-images.githubusercontent.com/11378447/66991152-cc677080-f0c7-11e9-9f23-63d66129bc6c.png)

**Comment by willianmano** [comment]
@giuliogatto Thanks for your reply. But this only occurs with your enviroment. Could you please give more details? Moodle version, Moove version, Non official installed Plugins.

**Comment by giuliogatto** [comment]
Hi @willianmano , thank you for your quick reply.
I investigated the issue and it turned out the problem were some custom CSS (in the RAW css settings of the theme).
I inherited the site and the theme so I wasn't aware that some custom CSS were added.
After your reply I double checked everything again and .. bang! The custom CSS was responsible for hiding the options in the Restrict Access window.
Thank you again for the great theme and the amount of work you put into it!
